hadoop-hdds/interface-client/src/main/proto/DatanodeClientProtocol.proto

```diff
@@ -100,6 +100,8 @@ enum Type {
   GetSmallFile = 16;
   CloseContainer = 17;
   GetCommittedBlockLength = 18;
+
+  StreamInit = 19;
```

Comment:
lets start this at 30.

Reply:
I tried to change this to 30, but I got an NPE exception when start datanode. Change this to 19 in order and it start normal.